ACPs/77-reinventing-subnets/README.md

```diff
@@ -109,21 +76,81 @@ type SubnetValidator struct {
+
+The `validationID` for validators added through `ConvertSubnetTx` is defined as the SHA256 hash of the 40 bytes resulting from concatenating the 32 byte `convertSubnetTxID` with the 8 byte `validatorIndex` (index in the `Validators` array within the transaction).
+
+The following serialization is defined as a `ValidatorData`:
+
+```text
++--------------+----------+-----------+
+|       nodeID : [32]byte |  32 bytes |
++--------------+----------+-----------+
+|       weight :   uint64 |   8 bytes |
++--------------+----------+-----------+
+| blsPublicKey : [48]byte |  48 bytes |
++--------------+----------+-----------+
+                          |  88 bytes |
+                          +-----------+
+```
+
+The following serialization is defined as the `SubnetConversionData`
+
+```text
++-------------------+--------------------------+-------------------------------------------------------+
+| convertSubnetTxID :                 [32]byte |                                              32 bytes |
++-------------------+--------------------------+-------------------------------------------------------+
+|    managerChainID :                 [32]byte |                                              32 bytes |
++-------------------+--------------------------+-------------------------------------------------------+
+|    managerAddress :                   []byte |                         4 + len(managerAddress) bytes |
++-------------------+--------------------------+-------------------------------------------------------+
+|        validators :          []ValidatorData |                        4 + len(validators) * 88 bytes |
++-------------------+--------------------------+-------------------------------------------------------+
+                                               | 72 + len(managerAddress) + len(validators) * 88 bytes |
+                                               +-------------------------------------------------------+
+```
+
+Once a `ConvertSubnetTx` is accepted, P-Chain validators must be willing to sign an `AddressedCall` with `sourceChainID` set to the P-Chain ID and the `sourceAddress` set to an empty byte array. The payload of the `AddressedCall` must be:
```

Comment:
With the rewording of this line we've lost the only reference to `SubnetConversionMessage` in this PR although it's still in the title of the PR and presumably will be referenced in the `avalanchego` code implementation. Should we add it back?

Reply:
I prefer to not name them in the ACP if it's not referenced again in the ACP (the other AddressedCall payloads don't have designated names)